insights/parsers/sys_module.py

```diff
@@ -0,0 +1,81 @@
+"""
+``/sys/module`` System Module Information
+=========================================
+
+A parser to parse the system module information.
+
+Parsers included in this module are:
+
+DMModUseBlkMq - file ``/sys/module/dm_mod/parameters/use_blk_mq``
+
+SCSIModUseBlkMq - file ``/sys/module/scsi_mod/parameters/use_blk_mq``
+------------------------------------------------------------
+
+"""
+
+
+from insights import parser, Parser
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+
+class XModUseBlkMq(Parser):
+    """
+    Parse for file `/sys/module/{dm_mod,scsi_mod}/parameters/use_blk_mq`.
+    File content shows if use_blk_mq parameter is on.
+
+    Sample Content::
+
+        Y - for on
+        N - for off
+    """
+
+    def parse_content(self, content):
+        if not content or len(content) != 1:
+            raise SkipException()
+        self.val = content[0].strip()
```

Comment:
Should be a private attribute since you have the `is_on` property?

Reply:
Question is answered in my above reply.